32 bit build with clang fails

```sh
export CC="clang"
export CFLAGS="-m32 -g -DWOLFSSL_AES_DIRECT -DWOLFSSL_STATIC_PSK -DHAVE_AES_ECB -DWOLFSSL_DES_ECB -DHAVE_ECC_SECPR2 -DHAVE_ECC_SECPR3 -DHAVE_ECC_BRAINPOOL -DHAVE_ECC_KOBLITZ"

git clone --depth 1 https://github.com/wolfSSL/wolfssl.git
git clone --depth 1 https://github.com/wolfSSL/wolfssh.git

cd wolfssl/
autoreconf -ivf
./configure --enable-static --disable-examples --disable-crypttests --disable-asm --enable-tls13 --enable-ocsp --enable-dtls --enable-sni --enable-blake2 --enable-blake2s --enable-curve25519 --enable-session-ticket --enable-nullcipher --enable-crl --enable-ed25519 --enable-psk --enable-earlydata --enable-postauth --enable-hrrcookie --enable-opensslextra --enable-certext --enable-tlsx --enable-oldtls --enable-tlsv10 --enable-indef --enable-psk --enable-ecccustcurves=all --enable-secure-renegotiation  --enable-curve25519 --enable-curve448 --enable-ed25519 --enable-ed448 --enable-ocspstapling --enable-srp --enable-ssh --enable-keygen
make -j$(nproc)
export LDFLAGS="-L$(realpath src/.libs/)"
export CFLAGS="$CFLAGS -I $(realpath .)"
cd ../

cd wolfssh/
autoreconf -ivf
./configure --enable-static --enable-all --disable-examples
make -j$(nproc)
```

This fails with the following error:

```
examples/sftpclient/sftpclient.c:1468:54: error: shift count >= width of type [-Werror,-Wshift-count-overflow]
                       (long)(((long)tmp->atrb.sz[1] << 32) | tmp->atrb.sz[0]));
```

Additionally, I think it shouldn't try to compile the examples when `./configure` was called with `--disable-examples`?

--- a/examples/sftpclient/include.am
+++ b/examples/sftpclient/include.am
@@ -1,10 +1,12 @@
 # vim:ft=automake
 # All paths should be given relative to the root
 
+if BUILD_EXAMPLE_CLIENTS
 if BUILD_SFTP
 noinst_PROGRAMS += examples/sftpclient/wolfsftp
 examples_sftpclient_wolfsftp_SOURCES      = examples/sftpclient/sftpclient.c \
                                             examples/sftpclient/sftpclient.h
 examples_sftpclient_wolfsftp_LDADD        = src/libwolfssh.la
 examples_sftpclient_wolfsftp_DEPENDENCIES = src/libwolfssh.la
+endif
 endif

--- a/examples/sftpclient/sftpclient.c
+++ b/examples/sftpclient/sftpclient.c
@@ -1464,8 +1464,8 @@ static int doCmds(func_args* args)
             while (tmp != NULL) {
                 if (WSTRNSTR(msg, "-s", MAX_CMD_SZ) != NULL) {
                     char tmpStr[WOLFSSH_MAX_FILENAME];
-                    XSNPRINTF(tmpStr, WOLFSSH_MAX_FILENAME, "%ld, ",
-                       (long)(((long)tmp->atrb.sz[1] << 32) | tmp->atrb.sz[0]));
+                    XSNPRINTF(tmpStr, WOLFSSH_MAX_FILENAME, "%lld, ",
+                       (long long)(((long long)tmp->atrb.sz[1] << 32) | tmp->atrb.sz[0]));
                     if (SFTP_FPUTS(args, tmpStr) < 0) {
                         err_msg("fputs error");
                         return -1;

